### ISSUE TITLE
:style directive

### ISSUE THREAD

**Issue by vanodevium** [dy/sprae#33]
Unfortunately, there is still an issue with сamelCase

```html
<header class="navbar is-flex p-0 pt-2 pb-2" :style="{ color: 'white', backgroundColor: '#df0000' }" >
...
</header>
```

**Comment by dy** [comment]
Hmm, try 10.9.3. I literally added your test case, it should have worked https://github.com/dy/sprae/blob/main/test/style.js#L48-L52

**Comment by vanodevium** [comment]
@dy 

It's a pity, but it's true: I conducted a lot of tests and realized that the UMD files are broken and do not work at all, even if you take it from the CDN and put it in the root of the project.

The problem is in the formatting of the file, I showed it in the screenshot #28 (yes, the [plugin](https://github.com/inqnuam/esbuild-plugin-umd-wrapper) is to blame).
I will try to send a request to the author of the plugin.

But for now it is necessary to come up with a temporary solution.
